app/Models/Signup.php

```diff
@@ -26,11 +26,11 @@ class Signup extends Model
     ];
 
     /**
-     * Each signup belongs to an event.
+     * Get the events associated with this signup.
      */
     public function event()
```

Comment:
I'm sorry this is kinda nitpicky, but this should be `events()` now that we're returning a collection. 😬